daemon/cmd/daemon.go

```diff
@@ -311,7 +311,7 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	if option.Config.ReadCNIConfiguration != "" {
 		netConf, err = cnitypes.ReadNetConf(option.Config.ReadCNIConfiguration)
 		if err != nil {
-			log.WithError(err).Fatal("Unable to read CNI configuration")
+			return nil, nil, fmt.Errorf("Unable to read CNI configuration: %v", err)
```

Comment:
We should be wrapping these errors rather than formatting them, right?

```suggestion
			return nil, nil, fmt.Errorf("Unable to read CNI configuration: %w", err)
```

Reply:
Yup you're right, muscle memory on that one. Ill do a pass and fix these.

Reply:
@joestringer fix via force push.